### PR TITLE
Run Only Unit Tests in Pre-Commit

### PR DESCRIPTION
**Pull request by crawfordleeds**
Per the conversation in https://github.com/mozilla/grcov/issues/308#issuecomment-519827076, the pre-commit hook should only run unit tests.

This PR changes the pre-commit configuration to only run unit tests and documents the option in the README for reference.

### CODE REVIEW

File: test/rust/main.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (marco-c) [inline test/rust/main.rs]: Why are this file and test/rust/src/main.rs necessary?

Comment (crawfordleeds) [reply test/rust/main.rs]: I'm not sure that they are necessary. They currently exist in the master branch so the existence of the files is a carryover of previous work but it looks like for some reason I added a couple blank lines. I don't recall actually modifying these files manually so I'm guessing the line was added by my IDE?

I just tried deleting these two files and the tests still pass.

I'll remove the blank lines and commit. Think we should remove these files in a new PR?

Comment (marco-c) [reply test/rust/main.rs]: We can leave them, I thought you were adding them now (the GitHub interface confused me).

Comment (crawfordleeds) [reply test/rust/main.rs]: Cool - I was confused at first too. I'm still somewhat confused how those changes ended up there in the first place 🙃

**Pull request merged**